examples/early-window-results/src/main/java/com/hazelcast/jet/examples/earlyresults/TradingVolumeOverTime.java

```diff
@@ -53,17 +53,18 @@
 
     private static final String VOLUME_LIST_NAME = "trading-volume";
     private static final int TRADES_PER_SEC = 3_000;
+    private static final int MAX_LAG = 5000;
     private static final int DURATION_SECONDS = 60;
 
     private static Pipeline buildPipeline() {
         Pipeline p = Pipeline.create();
-        p.readFrom(TradeGenerator.tradeSource(TRADES_PER_SEC, DURATION_SECONDS))
-         .withNativeTimestamps(TradeGenerator.MAX_LAG)
+        p.readFrom(TradeGenerator.tradeSource(1, TRADES_PER_SEC, MAX_LAG, DURATION_SECONDS))
+         .withNativeTimestamps(MAX_LAG)
          .window(tumbling(SECONDS.toMillis(2))
                  // comment out this line to see how the chart behaves without early results:
                  .setEarlyResultsPeriod(20)
          )
-         .aggregate(summingLong(trade -> trade.getQuantity() * trade.getPrice()))
```

Comment:
Is the change in the logic intended ?

Reply:
Yes. Since the original `TradeGenerators` emitted different kinds of trades I had to modify the examples to show the same TYPE of behaviour as before, even when moved to the new, common generator.